addons/l10n_sa/i18n_extra/l10n_sa.pot

```diff
@@ -1247,3 +1247,36 @@ msgstr ""
 #: model:account.tax.group,name:l10n_sa.sa_tax_group_taxes_15
 msgid "VAT Taxes"
 msgstr ""
+
+#. module: l10n_sa
+#: model:ir.model.fields,field_description:l10n_sa.field_account_bank_statement_line__l10n_sa_delivery_date
+#: model:ir.model.fields,field_description:l10n_sa.field_account_move__l10n_sa_delivery_date
+#: model:ir.model.fields,field_description:l10n_sa.field_account_payment__l10n_sa_delivery_date
+msgid "Supply Date"
```

Comment:
You need to translate all this in the `ar.po`. As we're in a l10n module, all this is not on Transifex